conda-forge.yml

```diff
@@ -1,7 +1,3 @@
-bot:
-  abi_migration_branches:
-  - v0.65
-  - 0.89.x
```

Comment:
@pkgw, @ocefpaf and @xhochy, any objections?

Reply:
I'm ok with that but my concern is problem the same the same as yours: long as gdal is ok these changes are fine. So let's wait for Uwe and Peter, that way we can have some diversity of opinions.

Reply:
This is fine by me too. As far as I know gdal was the only reason to keep the older branches active.

Reply:
I think uwe is kinda busy. You might want to merge without him.

Reply:
Thanks for letting us know @hmaarrfk.  I'll go ahead then.